apis/src/responses/tournament.rs

```diff
@@ -0,0 +1,104 @@
+use crate::responses::user::UserResponse;
+use chrono::{DateTime, Utc};
+use serde::{Deserialize, Serialize};
+use uuid::Uuid;
+
+#[derive(Clone, Serialize, Deserialize, Debug, PartialEq, Eq)]
+pub struct TournamentAbstractResponse {
```

Comment:
I dont like the name of this type, but I also dont have another suggestion for now XD

Reply:
If you do, let me know :) Naming is hard, and this one is the short `TournamentResponse` so I thought `Abstract` was fitting... but I am happy to change it!